### ISSUE TITLE
A11y_Windows Terminal_PowerShell_Screen reader: Single focus is provided to both 'Opening New Tab' and 'Dropdown' control and screen reader is reading as Split button which does not give much information about the control

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/terminal#5423]
GitHubTags: #A11y_WindowsTerminal; #WindowsTerminal; #A11yTCS; #A11yMAS; #Win10; #DesktopApp; #ScreenReader; #Narrator; #NVDA; #MAS1.3.1; #C+E-H; #A11ySev2;

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-tagott%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Windows Terminal
Version: 0.10.781.0
OS: Windows 10 (Version 1909, Build: 18363.778)

**Repro Steps:**
1.Turn on the screen reader
2.Hit the URL "aka.ms/terminal" and install "Windows terminal (Preview)" application
3.Open "Windows terminal (Preview)" application
4.Tab to '+' control present beside the tab
5.Verify whether every control is getting individual focus and screen reader is reading it as 'Open New tab' button and 'dropdown' 

**Actual Result:**
Single focus is provided to both 'Opening New Tab' and 'Dropdown' control and screen reader is reading as Split button which does not give much information about the control

**Expected Result:**
Different focus should be provided to the '+' symbol (Open New Tab) and 'Dropdown' and screen reader should read it as 'Open New Tab button' when focus is on '+' control and 'Dropdown' when focus is on dropdown control.

**User Impact:**
Visually disabled users will find difficulty, if the focus single focus is provided to two control and screen reader is reading as split button, user will get confused.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug 5423_Two controls getting single focus and screen reader is not clear information about the controls.pptx](https://github.com/microsoft/terminal/files/4503673/Bug.5423_Two.controls.getting.single.focus.and.screen.reader.is.not.clear.information.about.the.controls.pptx)

![Bug 5423_NVDA_Reading it as Split button](https://user-images.githubusercontent.com/51435266/79757617-f8df9c00-8339-11ea-8c03-3d14e607072e.png)

**Comment by Amulya-95** [comment]
The issue is resolved. Now the split button descriptive enough for the user to understand.
Hence closing the bug. Please find the attachments for reference.
 
[Bug_Split button is descriptive enough for user to understand.pptx](https://github.com/microsoft/terminal/files/4586351/Bug_Split.button.is.descriptive.enough.for.user.to.understand.pptx)

![Bug 5423_NVDA is reading descriptively about split button](https://user-images.githubusercontent.com/51435266/81170524-78b36a80-8fb8-11ea-8f82-5b7d6b7bf8ba.png)
